javascript/central-login-oidc/.env.example

```diff
@@ -0,0 +1,6 @@
+SCOPE=$SCOPE // For using "PINGONE" as the server type, the `revoke` scope must be set as well.
+TIMEOUT=$TIMEOUT
+WEB_OAUTH_CLIENT=$WEB_OAUTH_CLIENT
+NX_NO_CLOUD=true
```

Comment:
This one that's not really mentioned anywhere might be better down the bottom, and the way more important WELL_KNOWN variable moved higher up?

Reply:
Actually I have no clue what `NX_NO_CLOUD=true` does. If we don't need it we can remove it. @ryanbas21 @cerebrl any ideas?